src/styles/index.scss

```
@@ -1,3 +1,19 @@
+@use './mixins' as *;
+@use './variables' as *;
+@use './card' as *;
+@use './stars' as *;
+@use './price' as *;
+@use './button' as *;
+
 body {
+  @include center;
+  @include roboto400;
+
+  border: 1px;
+  border-color: #f3f3f3;
+  border-radius: 5px;
   margin: 0;
+  width: 198px;
+  height: 406px;
+  flex-direction: column;
```

body is not a container for card so these styles should be moved to card class 
```suggestion
  @include roboto400;
  margin: 0;
```